subsys/modbus/modbus_serial.c

```diff
@@ -568,7 +648,7 @@ int modbus_serial_init(struct modbus_context *ctx,
 		cfg->rtu_timeout = (numof_bits * if_delay_max) /
 				   param.serial.baud;
 	} else {
-		cfg->rtu_timeout = (numof_bits * if_delay_max) / 38400;
```

Comment:
Why and how is it relevant to your changes?

Reply:
Stupid mistake for that one, I will suppress this unnecessary change.